lib/bookingsync/api/client/rentals_contents_overrides.rb

```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module RentalsContentsOverrides
+      # List rentals_contents_overrides
+      #
+      # Returns all rentals contents overrides for the current account.
+      # @param options [Hash] A customizable set of options.
+      # @option options [Array] fields: List of fields to be fetched.
+      # @return [Array<BookingSync::API::Resource>] Array of rentals_contents_overrides.
+      #
+      # @example Get the list of rentals contents overrides for the current account
+      #   rentals_contents_overrides = @api.rentals_contents_overrides
+      #   rentals_contents_overrides.first.description.en # => "Overriden English description"
+      # @see http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/#list-rentals-contents-overrides
+      def rentals_contents_overrides(options = {}, &block)
+        paginate :rentals_contents_overrides, options, &block
```

Comment:
Nothing tests that these options are passed and usable and do stuff

Reply:
This is kind of tough to test, as according to docs it currently doesn't use any additional args.